crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -43,7 +48,11 @@ const SYNC_WINDOW: Duration = Duration::from_secs(3);
 pub type SyncStart = SetTimeout<()>;
 pub type SyncComplete = Timeout<()>;
 
-fan_in_message_type!(C8yMapperInput[MqttMessage, FsWatchEvent, SyncComplete] : Debug);
+pub(crate) type CmdId = String;
+pub(crate) type IdDownloadResult = (CmdId, DownloadResult);
```

Comment:
Putting `CmdId` in `DownloadRequest` doesn't hurt, but in `DownloadResult` is a problem since is `Result` type. If `Err`, difficult to retrieve `CmdId` although all consumer actors need to know `CmdId` in first.

```rust
pub type DownloadResult = Result<DownloadResponse, DownloadError>;
```